package.json

```diff
@@ -16,17 +16,19 @@
   },
   "devDependencies": {
     "babel-cli": "^6.6.5",
+    "babel-eslint": "^6.1.2",
+    "babel-plugin-istanbul": "^2.0.1",
     "babel-preset-es2015": "^6.6.0",
+    "eslint": "^3.5.0",
+    "eslint-config-defaults": "^9.0.0",
     "mocha": "^2.4.5",
+    "nyc": "^8.1.0",
     "sinon": "^1.17.5"
   },
-  "babel": {
-    "presets": [
-      "es2015"
-    ]
-  },
   "scripts": {
-    "test": "mocha --compilers js:babel-register --recursive",
-    "dist": "mkdir -p dist && rm -rf dist/* && babel lib --out-dir dist"
+    "lint": "eslint {lib,test}/**/*",
+    "test": "nyc mocha --compilers js:babel-register --recursive",
+    "dist": "babel lib/ --out-dir dist/",
+    "prepublish": "npm test && npm run lint && npm run dist"
```

Comment:
It runs whenever we run `npm publish` or `npm install` (locally, without arguments – `npm install foo-module` won't trigger it) [[source]](https://docs.npmjs.com/misc/scripts).

Whenever you have some time, I'd suggest to setup [semantic-release](https://github.com/semantic-release/semantic-release), it makes releasing and publishing a breeze. 😉